tracer/test/snapshots/CoverageRewriteTests.Rewritten.verified.txt

```diff
@@ -62,39 +63,50 @@ namespace CoverageRewriterAssembly
 				Console.WriteLine(MyProperty);
 				return;
 			}
-			scope.Report(2251838468915210uL, 2533330625560622uL);
+			counters[0]++;
+			counters[1]++;
 			Console.WriteLine("Main Method");
-			scope.Report(2814827077173275uL);
+			counters[2]++;
 			i = 0;
 			while (true)
 			{
-				scope.Report(2814874321813540uL);
+				counters[7]++;
 				if (i >= 100)
 				{
 					break;
 				}
-				scope.Report(3096280579112974uL, 3377772735758374uL);
+				counters[3]++;
+				counters[4]++;
 				Console.WriteLine(i);
-				scope.Report(3659230532665358uL, 2814912976519209uL);
+				counters[5]++;
+				counters[6]++;
 				i++;
 			}
-			scope.Report(4222180486217762uL);
+			counters[8]++;
 			MyProperty = "Value";
-			scope.Report(4785130439770155uL);
+			counters[9]++;
 			Console.WriteLine(MyProperty);
-			scope.Report(5066588236677130uL);
+			counters[10]++;
 		}
 
-		[AsyncStateMachine(typeof(<MainAsync>d__5))]
-		[DebuggerStepThrough]
-		public Task MainAsync()
+		public async Task MainAsync()
 		{
-			<MainAsync>d__5 stateMachine = new <MainAsync>d__5();
-			stateMachine.<>t__builder = AsyncTaskMethodBuilder.Create();
-			stateMachine.<>4__this = this;
-			stateMachine.<>1__state = -1;
-			stateMachine.<>t__builder.Start(ref stateMachine);
-			return stateMachine.<>t__builder.Task;
+			await Task.Delay(100);
+		}
+	}
+}
+namespace Datadog.Trace.Ci.Coverage.Metadata.Target
+{
+	public class ModuleCoverage : ModuleCoverageMetadata
+	{
+		public ModuleCoverage()
+		{
+			Metadata = new int[2][];
+			Metadata[0] = new int[0];
```

Comment:
I guess this could use `Array.Empty<int>()`?

Reply:
Sure, now I have to emit it in IL :D

Reply:
done in [99e3387](https://github.com/DataDog/dd-trace-dotnet/pull/3134/commits/99e3387d01f21ef3fff163365cdfccf2c0f5a15c)